distance_cost_matrix: [
  [-1.000000, 35.000000, 36.000000, 37.000000, 33.000000, 34.000000, 32.000000, 0.000000, 1.000000, 2.000000, 3.000000, 4.000000, 5.000000, 6.000000, 11.000000, 12.000000, 13.000000, 14.000000, 15.000000, 16.000000, 17.000000, 18.000000, 19.000000, 20.000000, 23.000000, 24.000000, 25.000000, 26.000000, 27.000000, 28.000000, 29.000000, 30.000000],
  [35.000000, -1.000000, -1.000000, -1.000000, -1.000000, -1.000000, -1.000000, 585.189392, 507.108673, 481.750580, 509.399384, 588.263916, 602.754150, 714.322937, 571.483582, 459.955292, 367.760651, 502.410645, 636.638367, 496.608185, 401.337555, 305.511200, 216.908142, 153.665756, 147.016861, 226.387939, 324.020691, 421.624542, 521.519775, 623.598206, 707.699707, 743.748413],
  [36.000000, -1.000000, -1.000000, -1.000000, -1.000000, -1.000000, -1.000000, 590.800659, 515.338745, 492.204010, 520.839661, 599.707397, 614.135254, 775.966919, 582.395935, 475.713318, 383.859253, 495.575531, 629.803223, 507.911804, 412.519836, 316.430786, 226.899216, 163.748276, 138.718094, 216.085526, 313.071411, 410.427948, 510.195648, 612.205566, 696.249695, 732.659302],
  [37.000000, -1.000000, -1.000000, -1.000000, -1.000000, -1.000000, -1.000000, 588.629089, 511.242950, 486.272247, 513.834717, 592.247314, 606.660156, 718.285461, 574.966614, 463.065765, 370.648041, 498.717804, 632.945557, 500.430420, 405.046600, 309.005371, 219.978394, 156.612656, 142.838074, 221.882950, 319.489685, 417.114563, 517.036255, 619.137085, 703.272034, 739.222595],
  [33.000000, -1.000000, -1.000000, -1.000000, -1.000000, -1.000000, -1.000000, 590.529297, 515.031128, 491.885529, 520.540955, 599.451599, 613.886047, 775.773804, 582.181458, 475.520203, 383.661285, 495.845856, 630.073547, 507.669617, 412.287079, 316.215363, 226.717026, 163.571884, 139.026657, 216.407654, 313.388855, 410.740265, 510.503662, 612.510437, 696.550903, 732.974060],
  [34.000000, -1.000000, -1.000000, -1.000000, -1.000000, -1.000000, -1.000000, 588.395142, 510.974670, 485.991333, 513.569092, 592.018372, 606.436951, 718.057861, 574.773682, 462.898987, 370.495911, 498.963806, 633.191528, 500.213257, 404.837677, 308.811676, 219.813858, 156.455826, 143.110382, 222.165695, 319.768921, 417.389801, 517.308167, 619.406494, 703.538391, 739.500122],
  [32.000000, -1.000000, -1.000000, -1.000000, -1.000000, -1.000000, -1.000000, 585.011658, 506.888000, 481.502411, 509.150665, 588.035461, 602.529480, 714.099304, 571.280273, 459.757538, 367.519165, 502.470459, 636.698181, 496.387695, 401.119537, 305.298828, 216.708557, 153.446594, 147.187546, 226.594193, 324.238678, 421.847015, 521.744385, 623.824097, 707.921509, 743.971008],
  [0.000000, 585.189392, 590.800659, 588.629089, 590.529297, 588.395142, 585.011658, -1.000000, 173.869827, 343.248260, 494.468842, 669.775696, 694.944580, 804.647583, 750.440674, 757.477234, 817.497192, 986.118103, 1165.438721, 652.172058, 625.584412, 614.500366, 619.026306, 638.223999, 723.635559, 779.330200, 849.623230, 923.680115, 1002.790771, 1087.413696, 1154.259155, 1227.135742],
  [1.000000, 507.108673, 515.338745, 511.242950, 515.031128, 510.974670, 506.888000, 173.869827, -1.000000, 169.378479, 320.599701, 495.905884, 521.074768, 630.784302, 578.093262, 596.430054, 681.140381, 858.851379, 1044.994263, 480.882690, 461.977631, 464.626495, 486.903870, 526.164856, 652.005615, 723.377197, 808.361023, 894.445129, 983.961121, 1077.340088, 1151.960205, 1211.627319],
  [2.000000, 481.750580, 492.204010, 486.272247, 491.885529, 485.991333, 481.502411, 343.248260, 169.378479, -1.000000, 151.224457, 326.527466, 351.696320, 461.418793, 411.403198, 448.763550, 568.238953, 752.899353, 943.705322, 316.692780, 313.331848, 341.157684, 390.863831, 455.806305, 623.965332, 706.621033, 802.947632, 897.858398, 994.790710, 1094.196411, 1174.595703, 1220.249023],
  [3.000000, 509.399384, 520.839661, 513.834717, 520.540955, 513.569092, 509.150665, 494.468842, 320.599701, 151.224457, -1.000000, 175.311539, 200.482346, 310.238678, 265.922089, 335.806213, 496.689148, 681.602661, 872.574768, 179.228912, 210.385590, 277.095673, 356.313782, 442.443512, 658.771851, 728.300720, 829.215942, 928.720215, 1029.514282, 1131.971191, 1215.854980, 1249.199829],
  [4.000000, 588.263916, 599.707397, 592.247314, 599.451599, 592.018372, 588.035461, 669.775696, 495.905884, 326.527466, 175.311539, -1.000000, 25.171684, 138.318939, 115.235435, 256.868134, 461.803619, 632.517029, 815.364441, 99.223175, 195.133057, 294.509796, 390.474182, 486.668671, 746.383789, 847.222656, 961.833923, 1068.355347, 1181.557983, 1295.483643, 1382.915405, 1300.893066],
  [5.000000, 602.754150, 614.135254, 606.660156, 613.886047, 606.436951, 602.529480, 694.944580, 521.074768, 351.696320, 200.482346, 25.171684, -1.000000, 113.308311, 100.201492, 253.370880, 461.844727, 628.953003, 809.718506, 107.934227, 205.312851, 305.106354, 401.271942, 497.588135, 756.598633, 857.432495, 972.040588, 1078.557617, 1191.759644, 1305.682129, 1393.111084, 1309.920898],
  [6.000000, 714.322937, 775.966919, 718.285461, 775.773804, 718.057861, 714.099304, 804.647583, 630.784302, 461.418793, 310.238678, 138.318939, 113.308311, -1.000000, 192.211472, 359.143646, 570.633667, 732.639038, 910.261292, 220.980789, 318.400177, 418.212677, 514.387451, 610.709229, 879.478760, 980.318542, 1094.935913, 1201.454102, 1314.664551, 1428.588989, 1518.266724, 1427.639038],
  [11.000000, 571.483582, 582.395935, 574.966614, 582.181458, 574.773682, 571.280273, 750.440674, 578.093262, 411.403198, 265.922089, 115.235435, 100.201492, 192.211472, -1.000000, 169.411301, 381.690643, 539.249451, 715.917847, 100.747185, 174.726822, 266.280640, 358.694977, 452.822632, 706.967712, 746.809753, 850.314575, 950.763550, 1052.577271, 1155.631958, 1242.405884, 1251.818848],
  [12.000000, 459.955292, 475.713318, 463.065765, 475.520203, 462.898987, 459.757538, 757.477234, 596.430054, 448.763550, 335.806213, 256.868134, 253.370880, 359.143646, 169.411301, -1.000000, 212.443726, 375.827667, 559.059143, 171.771774, 136.004639, 163.119354, 230.818634, 313.892395, 504.133514, 595.200500, 696.716614, 795.789856, 896.614380, 998.823486, 1085.543335, 1089.569946],
  [13.000000, 367.760651, 383.859253, 370.648041, 383.661285, 370.495911, 367.519165, 817.497192, 681.140381, 568.238953, 496.689148, 461.803619, 461.844727, 570.633667, 381.690643, 212.443726, -1.000000, 185.884995, 377.437042, 367.113861, 290.414429, 227.870697, 198.865967, 213.757477, 343.907715, 423.869110, 517.599243, 611.772217, 709.249451, 808.867554, 894.956543, 889.562073],
  [14.000000, 502.410645, 495.575531, 498.717804, 495.845856, 498.963806, 502.470459, 986.118103, 858.851379, 752.899353, 681.602661, 632.517029, 628.953003, 732.639038, 539.249451, 375.827667, 185.884995, -1.000000, 191.552094, 542.631470, 473.348907, 413.346802, 372.093292, 352.904938, 392.010986, 441.029266, 509.638580, 586.416992, 671.194885, 760.914185, 842.708191, 814.817017],
  [15.000000, 636.638367, 629.803223, 632.945557, 630.073547, 633.191528, 636.698181, 1165.438721, 1044.994263, 943.705322, 872.574768, 815.364441, 809.718506, 910.261292, 715.917847, 559.059143, 377.437042, 191.552094, -1.000000, 728.865845, 663.562744, 604.744934, 559.199768, 527.400024, 512.406555, 531.257080, 569.003113, 620.535522, 684.749146, 757.600281, 830.158142, 777.943115],
  [16.000000, 496.608185, 507.911804, 500.430420, 507.669617, 500.213257, 496.387695, 652.172058, 480.882690, 316.692780, 179.228912, 99.223175, 107.934227, 220.980789, 100.747185, 171.771774, 367.113861, 542.631470, 728.865845, -1.000000, 97.439896, 197.250534, 293.425140, 389.746979, 640.606506, 741.447571, 856.058899, 962.583191, 1075.784546, 1189.712158, 1277.218750, 1202.633789],
  [17.000000, 401.337555, 412.519836, 405.046600, 412.287079, 404.837677, 401.119537, 625.584412, 461.977631, 313.331848, 210.385590, 195.133057, 205.312851, 318.400177, 174.726822, 136.004639, 290.414429, 473.348907, 663.562744, 97.439896, -1.000000, 99.813713, 195.989227, 292.311523, 534.933167, 635.775330, 750.382507, 856.911194, 970.106445, 1084.036499, 1171.554688, 1105.887451],
  [18.000000, 305.511200, 316.430786, 309.005371, 316.215363, 308.811676, 305.298828, 614.500366, 464.626495, 341.157684, 277.095673, 294.509796, 305.106354, 418.212677, 266.280640, 163.119354, 227.870697, 413.346802, 604.744934, 197.250534, 99.813713, -1.000000, 96.175522, 192.497818, 426.657104, 527.499390, 642.099487, 748.633423, 861.820190, 975.752808, 1063.285278, 1006.778564],
  [19.000000, 216.908142, 226.899216, 219.978394, 226.717026, 219.813858, 216.708557, 619.026306, 486.903870, 390.863831, 356.313782, 390.474182, 401.271942, 514.387451, 358.694977, 230.818634, 198.865967, 372.093292, 559.199768, 293.425140, 195.989227, 96.175522, -1.000000, 96.322289, 321.838379, 422.669800, 537.250793, 643.787842, 756.959351, 870.892578, 958.441162, 911.430176],
  [20.000000, 153.665756, 163.748276, 156.612656, 163.571884, 156.455826, 153.446594, 638.223999, 526.164856, 455.806305, 442.443512, 486.668671, 497.588135, 610.709229, 452.822632, 313.892395, 213.757477, 352.904938, 527.400024, 389.746979, 292.311523, 192.497818, 96.322289, -1.000000, 216.961044, 317.753235, 432.292053, 538.829285, 651.975769, 765.907776, 853.475769, 865.118103],
  [23.000000, 147.016861, 138.718094, 142.838074, 139.026657, 143.110382, 147.187546, 723.635559, 652.005615, 623.965332, 658.771851, 746.383789, 756.598633, 879.478760, 706.967712, 504.133514, 343.907715, 392.010986, 512.406555, 640.606506, 534.933167, 426.657104, 321.838379, 216.961044, -1.000000, 94.528503, 199.006470, 300.164459, 402.457825, 506.007965, 592.280518, 615.185120],
  [24.000000, 226.387939, 216.085526, 221.882950, 216.407654, 222.165695, 226.594193, 779.330200, 723.377197, 706.621033, 728.300720, 847.222656, 857.432495, 980.318542, 746.809753, 595.200500, 423.869110, 441.029266, 531.257080, 741.447571, 635.775330, 527.499390, 422.669800, 317.753235, 94.528503, -1.000000, 104.480179, 205.638062, 307.930023, 411.480469, 497.779846, 522.249878],
  [25.000000, 324.020691, 313.071411, 319.489685, 313.388855, 319.768921, 324.238678, 849.623230, 808.361023, 802.947632, 829.215942, 961.833923, 972.040588, 1094.935913, 850.314575, 696.716614, 517.599243, 509.638580, 569.003113, 856.058899, 750.382507, 642.099487, 537.250793, 432.292053, 199.006470, 104.480179, -1.000000, 101.158081, 203.451889, 307.001617, 393.373474, 420.122955],
  [26.000000, 421.624542, 410.427948, 417.114563, 410.740265, 417.389801, 421.847015, 923.680115, 894.445129, 897.858398, 928.720215, 1068.355347, 1078.557617, 1201.454102, 950.763550, 795.789856, 611.772217, 586.416992, 620.535522, 962.583191, 856.911194, 748.633423, 643.787842, 538.829285, 300.164459, 205.638062, 101.158081, -1.000000, 102.296227, 205.844254, 292.324829, 322.768097],
  [27.000000, 521.519775, 510.195648, 517.036255, 510.503662, 517.308167, 521.744385, 1002.790771, 983.961121, 994.790710, 1029.514282, 1181.557983, 1191.759644, 1314.664551, 1052.577271, 896.614380, 709.249451, 671.194885, 684.749146, 1075.784546, 970.106445, 861.820190, 756.959351, 651.975769, 402.457825, 307.930023, 203.451889, 102.296227, -1.000000, 103.550751, 190.201019, 228.154373],
  [28.000000, 623.598206, 612.205566, 619.137085, 612.510437, 619.406494, 623.824097, 1087.413696, 1077.340088, 1094.196411, 1131.971191, 1295.483643, 1305.682129, 1428.588989, 1155.631958, 998.823486, 808.867554, 760.914185, 757.600281, 1189.712158, 1084.036499, 975.752808, 870.892578, 765.907776, 506.007965, 411.480469, 307.001617, 205.844254, 103.550751, -1.000000, 87.363144, 142.798203],
  [29.000000, 707.699707, 696.249695, 703.272034, 696.550903, 703.538391, 707.921509, 1154.259155, 1151.960205, 1174.595703, 1215.854980, 1382.915405, 1393.111084, 1518.266724, 1242.405884, 1085.543335, 894.956543, 842.708191, 830.158142, 1277.218750, 1171.554688, 1063.285278, 958.441162, 853.475769, 592.280518, 497.779846, 393.373474, 292.324829, 190.201019, 87.363144, -1.000000, 112.133392],
  [30.000000, 743.748413, 732.659302, 739.222595, 732.974060, 739.500122, 743.971008, 1227.135742, 1211.627319, 1220.249023, 1249.199829, 1300.893066, 1309.920898, 1427.639038, 1251.818848, 1089.569946, 889.562073, 814.817017, 777.943115, 1202.633789, 1105.887451, 1006.778564, 911.430176, 865.118103, 615.185120, 522.249878, 420.122955, 322.768097, 228.154373, 142.798203, 112.133392, -1.000000],
]